From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@thenewwazoo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@cherylmc - Could you please review this proposed update from @thenewwazoo in the public repo?

tysonn commented:
@cherylmc - Second notice, please review

cherylmc commented:
Hi! I made a small change to the wording and patched this. If that looks OK, it's fine to merge this in.

thenewwazoo commented:
lgtm

cherylmc commented:
Hi @thenewwazoo -I've pulled in the change from your branch and added it to a PR that was just merged in. So we're good to go! Thanks for brining this to our attention. @tysonn - the other PR was just merged in, so it's OK to close this one.

BryanLa committed:
updates

iainfoulds committed:
[VMSS] Updating phrasing

iainfoulds committed:
[VMSS] Updating autoscale PowerShell doc

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

djlan committed:
Update HDInsight VSCode document

BryanLa committed:
updates

iainfoulds committed:
[VMSS] Updating for new custom image experience

v-alje committed:
Bringing even with master.

iainfoulds committed:
[VMSS] Edits, date updates, code formatting

iainfoulds committed:
[VMSS] Aligning release dates

rjagiewich committed:
Merge pull request #35148 from iainfoulds/vmssmvcfinaledits

 [VMSS] Final edits

Commit:
Simplified Image Tutorial

v-alje committed:
Bringing even with master.

Commit:
updating the doc for creating vms from images

djlan committed:
Update HDInsight IntelliJ and Eclipse document for link cluster support

v-alje committed:
Bringing even with master.

Commit:
update Azure virtual machine scale sets LP to Azure MVC format

Commit:
update author to fix warning error

v-alje committed:
Bringing even with master.

Commit:
update link to index.yml

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

BryanLa committed:
Merge branch 'master' of https://www.github.com/microsoftdocs/azure-docs-pr into bryanla-dps-concepts-autoprovisioning

BryanLa committed:
final draft

BryanLa committed:
formatting updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

BryanLa committed:
pre-review tweaks

BryanLa committed:
compress table more

BryanLa committed:
compress table more

Commit:
add missing word "you"

BryanLa committed:
2nd review updates

v-alje committed:
Bringing even with master.

iainfoulds committed:
[VMSS} Updating for disk support

iainfoulds committed:
[VMSS} Update LUN numbering

BryanLa committed:
2nd review updates

rmca14 committed:
Merge pull request #34454 from iainfoulds/vmssmvcnewpowershellexperience

[VMSS] Updating for new PowerShell cmdlet experience

iainfoulds committed:
[VMSS] Cleaning up for MVC naming and retirements

iainfoulds committed:
[VMSS] Cleaning up and removing duplicate information

iainfoulds committed:
[VMSS] Resolving validation errors on broken links

iainfoulds committed:
[VMSS] Removing duplicate info, linking to tutorials

v-alje committed:
Bringing even with master.

v-geberr committed:
concept entities regex

v-geberr committed:
regex entity max item count

v-geberr committed:
regex entity data extraction

iainfoulds committed:
[VMSS] Updating article ownership

PRMerger12 committed:
Merge pull request #35907 from iainfoulds/patch-20

[VMSS] Updating article ownership

v-geberr committed:
regex entity on utterances how-to page

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

iainfoulds committed:
[VMSS] Merging update doc ownership of deleted file from PR #35907

iainfoulds committed:
[VMSS] Retiring old document

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

v-alje committed:
Bringing even with master.

v-geberr committed:
edit date

iainfoulds committed:
[VM] Updating VM name in final commands

v-geberr committed:
regex howto entity

v-kents committed:
add index.md back for merge error

v-geberr committed:
character max

v-geberr committed:
removed text about ignoring case - not sure what LUIS is doing

v-geberr committed:
edits

v-alje committed:
Bringing even with master.

ktoliver committed:
Merge pull request #35369 from v-savila/1207806upd

update Azure virtual machine scale sets LP to Azure MVC format

v-geberr committed:
confirmation from Carol about where regex is applied in pipeline

carolinacmoravia committed:
L14522: Extra square bracket in link structure

Hello, @syntaxc4, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

iainfoulds committed:
Merge branch 'release-mvc-vmss' of https://github.com/Microsoft/azure-docs-pr into vmssmvccleanup

itechedit committed:
Merge pull request #35847 from iainfoulds/vmssmvccleanup

[VMSS] MVC cleanup and final tasks

v-geberr committed:
added links and clarified text

v-geberr committed:
added links

v-alje committed:
Bringing even with master.

BryanLa committed:
post review updates

Commit:
updated examples

Commit:
slight wording change

cynthn committed:
Merge pull request #9 from danielsollondon/patch-2

slight wording change

cynthn committed:
Merge pull request #8 from danielsollondon/patch-1

updated examples

BryanLa committed:
post review updates

Commit:
clean-up before publishing

BryanLa committed:
post review updates

v-alje committed:
Bringing even with master.

Commit:
PS version

MSSedusch committed:
SAP: update SAP HANA SR guide

djlan committed:
Update png files

MSSedusch committed:
add missing overview image

dominicbetts committed:
Remove redundant snippet

MSSedusch committed:
update port, reorder deploy

v-geberr committed:
regex entity quickstart

v-geberr committed:
fixing broken links

v-geberr committed:
fix link

BryanLa committed:
updates

j-martens committed:
AML: redirecting release notes to single article

j-martens committed:
AML: formatting fix

j-martens committed:
AML: creating a single release note file

j-martens committed:
AML: release notes updates

BryanLa committed:
updates

v-geberr committed:
working thru qs details

j-martens committed:
AML: Move release note images into one dir

j-martens committed:
AML: Split the AML overview into 2 / other ML product file

bwren committed:
Minor grammar fix

j-martens committed:
AML: updating metadata

v-alje committed:
Bringing even with master.

iainfoulds committed:
[VMSS] Updating sample scripts for new PowerShell experience

j-martens committed:
AML: rename Release notes to What's new

Commit:
changes to perf counters with wad

iainfoulds committed:
Merge branch 'release-mvc-vmss' of https://github.com/Microsoft/azure-docs-pr into vmssmvcscriptsamplecleanup

Commit:
adding topic

j-martens committed:
AML: Clean up release note formatting

j-martens committed:
AML: Article title

v-geberr committed:
edits

Commit:
PR feedback

MikeRayMSFT committed:
Stage new image with corrected resource name

esung22 committed:
ADLS security doc updates

ChristianWolf42 committed:
Updating geo-dr documentation slightly

BryanLa committed:
updates from Nicole's feedback

douglaslMS committed:
Fixed a typo pointed out by CSS.

Commit:
upated author

iainfoulds committed:
[VM] Updating command example to use secured password

mmacy committed:
[ACI] update Node.js ref links to new target

Commit:
Merge pull request #36300 from douglaslMS/adf-fix-typo

Fixed a typo pointed out by CSS.

j-martens committed:
AML: trying a workaround to review block

billmath committed:
fixing url

rmca14 committed:
Merge pull request #35114 from BryanLa/bryanla-dps-concepts-autoprovisioning

DPS auto-provisioning concepts

RenaShahMSFT committed:
Adding Clarification on persistence of connection

neilpeterson committed:
[ACS] removing script / placing back commands

mmacy committed:
[ACR] H1 and landing page consistency pass

iainfoulds committed:
[VMSS] Removing redundant index.md file

ktoliver committed:
Indent lines to maintain numbering of procedure steps

Indent lines to maintain numbering of procedure steps

PRMerger19 committed:
Merge pull request #36294 from ChristianWolf42/OptimizeGeoDrDocs

Updating geo-dr documentation slightly

v-geberr committed:
new code

ktoliver committed:
Merge pull request #34905 from iainfoulds/vmnewvmcmdlets

[VM] Updates for New-AzureRmVm

ktoliver committed:
Merge pull request #36293 from srrengar/sravworkbranch

Changes to perf counters with wad

Commit:
Merge pull request #36309 from RenaShahMSFT/Persistence

Adding Clarification on persistence of connection

ktoliver committed:
Merge pull request #36287 from MikeRayMSFT/20180326_ImageNameFromCustomerSubmission

Stage new image with corrected resource name

rjagiewich committed:
Merge pull request #36295 from esung22/updates

ADLS security doc updates

ktoliver committed:
Merge pull request #36303 from mmacy/aci-fix-nodejs-toc

[ACI] update Node.js ref links to new target

PRMerger10 committed:
Merge pull request #36259 from bwren/update-management

Update Management minor grammar fix

HeidiSteen committed:
break out indexers into own section

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5869

Updating default disk number limitation.

johnkemnetz committed:
Update monitoring-enable-diagnostic-logs-using-template.md

ktoliver committed:
Merge pull request #36310 from neilpeterson/osba-zsh-patch

[ACS] removing script / placing back commands

PRMerger17 committed:
Merge pull request #36321 from iainfoulds/patch-24

[VM] Fixes MicrosoftDocs/azure-docs#5869

v-alje committed:
Bringing even with master.

rjagiewich committed:
Merge pull request #36312 from mmacy/acr-consistency-pass

[ACR] H1 and landing page consistency pass

PRMerger18 committed:
Merge pull request #36324 from johnkemnetz/patch-2

clarify setting name

ktoliver committed:
Merge pull request #36265 from iainfoulds/vmssmvcscriptsamplecleanup

[VMSS] Updating sample scripts for new PowerShell experience

wesmc7777 committed:
User Story 1016474 clarify clustering affect on max connections

wesmc7777 committed:
fix typo

j-martens committed:
AML: Fixing other non-blocking warnings raised by Ross

j-martens committed:
Merge branch 'master' of https://github.com/j-martens/azure-docs-pr

* 'master' of https://github.com/j-martens/azure-docs-pr:
  AML: trying a workaround to review block

christiankuhtz committed:
tweak

PRMerger19 committed:
Merge pull request #36330 from wesmc7777/redis_updates

User Story 1016474 clarify clustering affect on max connections

Commit:
Merge pull request #36314 from v-geberr/0326-fix-csharp-code-for-timeoutOffset-from-amir

new code

v-geberr committed:
regular expression instead of regex

v-geberr committed:
regular expression instead of regex

v-geberr committed:
regular expression instead of regex

v-geberr committed:
regular expression instead of regex

v-geberr committed:
regular expression instead of regex

v-geberr committed:
regular expression instead of regex

esung22 committed:
minor update, fixing some issues with last PR

chaosrealm committed:
Fixing a stray comma in sample JSON 

Fix #6271

chaosrealm committed:
Fix a back-to-the-future date

Fix for #6305

PRMerger16 committed:
Merge pull request #36336 from chaosrealm/patch-5

Fixing a stray comma in sample JSON

PRMerger19 committed:
Merge pull request #36337 from chaosrealm/patch-6

Fix a back-to-the-future date

GitHubber17 committed:
Merge pull request #36335 from esung22/updates

minor update, fixing some issues with last PR

j-martens committed:
AML: fixing blocking issues

Commit:
Upload doc

Commit:
Upload image updates for simplified

Commit:
Reverting file to master

Commit:
Cloud Shell

zhiweiwangmsft committed:
Add risky IP permissions

zhiweiwangmsft committed:
Add new line

Commit:
Merge pull request #36345 from zhiweiwangmsft/patch-25

Add risky IP permissions

yanancai committed:
add installation for VS 2017

femsulu committed:
Issue #6302 - fix misspelling

Fixed typo in Virtualization (misspelled as Virtualiztion)

craigshoemaker committed:
Change ownership from tamram to craigshoemaker

PRMerger12 committed:
Merge pull request #36348 from femsulu/patch-5

(Forums Team) Issue #6302 - fix misspelling

rayne-wiselman committed:
fixing public PR issue

PRMerger12 committed:
Merge pull request #36226 from dominicbetts/pyhton-javascript

Remove redundant snippet

v-geberr committed:
edits

MSSedusch committed:
add HANA 2.0 description

MSSedusch committed:
fix issues from PR 35762

PRMerger18 committed:
Merge pull request #36014 from carolinacmoravia/patch-653

L14522: Extra square bracket in link structure

v-geberr committed:
new qs, rename qs

Commit:
Merge pull request #36351 from rayne-wiselman/rayne-march27

fixing public PR issue

PRMerger12 committed:
Merge pull request #36349 from craigshoemaker/crs-ownership-reassign

Change ownership from tamram to craigshoemaker

MSSedusch committed:
fix hostnames

MSSedusch committed:
fix spelling erros

v-geberr committed:
change ms.date

v-geberr committed:
touch

PRMerger15 committed:
Merge pull request #36331 from christiankuhtz/patch-107

tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

v-geberr committed:
per Omar Elhariry

mimig1 committed:
Cosmos DB: Removed mention of docdbswat dl.

v-geberr committed:
per Omar Elhariry

PRMerger18 committed:
Merge pull request #35897 from v-geberr/0321-regex-limits

regex entity max item count

PRMerger18 committed:
Merge pull request #35902 from v-geberr/0321-concept-extracting-regex-data

regex entity data extraction

PRMerger18 committed:
Merge pull request #35896 from v-geberr/0321-concept-entity-regex

regex concept entities

v-geberr committed:
fix link

rmca14 committed:
Merge pull request #35265 from cynthn/simple-image

Simplified Image Tutorial

ktoliver committed:
Merge pull request #35928 from v-geberr/0321-regex-howto-utterances

regex howto utterances

itechedit committed:
Merge pull request #35176 from djlan/master

Update HDInsight VSCode document

rmca14 committed:
Merge pull request #35942 from v-geberr/0321-regex-howto-entities

regex howto entity

vhorne committed:
A change from PubPR 4521 plus other changes

rmca14 committed:
Merge pull request #36252 from j-martens/master

AML: redirecting release notes to single article

ktoliver committed:
Merge pull request #36201 from MSSedusch/sap-hana

SAP: update SAP HANA SR guide

v-geberr committed:
edits

Commit:
Updating the SQL DW capacity limits

rmca14 committed:
Merge pull request #36339 from cynthn/image-upload

Upload image updates for simplified

PRMerger17 committed:
Merge pull request #36364 from vhorne/pubpr4521

A change from PubPR 4521 plus other changes

Commit:
Updating the string literals

itechedit committed:
Merge pull request #36356 from v-geberr/0327-quickstart-title-change

edits

rmca14 committed:
Merge pull request #36346 from yanancai/master

Change ADL tools installation for VS2017

Commit:
Fixed bad link

itechedit committed:
Merge pull request #36231 from v-geberr/0326-quickstart-regex-entity

regex entity quickstart

PRMerger20 committed:
Merge pull request #36359 from christiankuhtz/patch-108

Tweak

Commit:
cloud shell

Commit:
Merge pull request #36365 from TwoUnder/master

Updating the SQL DW capacity limits

gsilva5 committed:
Updated GA regions

Commit:
Merge pull request #36366 from jeffgilb/327

Fixed bad link

itechedit committed:
Merge pull request #36357 from v-geberr/0327-landing-page

new qs, rename qs

Commit:
format

iainfoulds committed:
[VM] Updating for latest Jenkins installer

v-shils committed:
Merge pull request #36322 from HeidiSteen/heidist-master

break out indexers into own section

PRMerger14 committed:
Merge pull request #36370 from iainfoulds/vmjenkinsupdate

[VM] Updating for latest Jenkins installer

Commit:
add a note

iainfoulds committed:
[VM] Updating author on unmaintained docs

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Merge pull request #32859 from cynthn/32193

Daniel's PR 32193 review

PRMerger18 committed:
Merge pull request #36372 from shizn/master

add a note for iot edge python module

PRMerger18 committed:
Merge pull request #36374 from iainfoulds/vmauthorupdates

[VM] Updating author on unmaintained docs

v-alje committed:
Merge pull request #36376 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #36305 from billmath/qlik

fixing url

christiankuhtz committed:
tweak

christiankuhtz committed:
tweak

Commit:
Merge pull request #36363 from mimig1/03272018removedl

Cosmos DB: Removed mention of docdbswat dl.

v-alje committed:
Merge pull request #33477 from MicrosoftDocs/release-mvc-vmss

[VMSS] DO NOT MERGE - MVC VMSS staging build

Commit:
Merge pull request #36380 from christiankuhtz/patch-110

tweak

PRMerger10 committed:
Merge pull request #36360 from christiankuhtz/patch-109

Tweak

PRMerger10 committed:
Merge pull request #36381 from christiankuhtz/patch-111

tweak

PRMerger12 committed:
Merge pull request #36369 from gsilva5/patch-16

Updated GA regions

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
